plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/propertysheets/AbstractTwoColumnEditorPropertySection.java

```diff
@@ -57,4 +62,11 @@ protected Layout createRightColumnLayout() {
 		return createBodyLayout();
 	}
 

```

Comment:
I am not sure if the createDocumentationControl should be part of the AbstractTwoColumnEditorPropertySection. Not every TwoColumnEditorPropertySection needs a DocumentationControl.  Previous version however had duplicate code so this is fine for me.